src/device/device.ts

```diff
@@ -513,6 +518,12 @@ export class MicrobitWebUSBConnection
     }
   };
 
+  clearDevice(): void {
+    this.connection = undefined;
+    this.device = undefined;
```

Comment:
What should this do if you're connected? Make it async and disconnect first? 

We plan to extract this component at some point so its API needs to be reasonably defensive.

Reply:
[`ebdc644`](https://github.com/microbit-foundation/python-editor-next/pull/702/commits/ebdc644bd0769de357211510a6da495abc6f207b)